pkg/sfu/buffer/buffer.go

```diff
@@ -69,7 +69,7 @@ type Buffer struct {
 	minPacketProbe     int
 	lastPacketRead     int
 	bitrate            atomic.Value
-	bitrateHelper      [4]uint64
+	bitrateHelper      [4]int64
```

Comment:
Making bandwidth related vars `int64` as I am doing some delta calculations and the delta can go negative. So, changing type uniformly rather than casting.